ext/action-popup.html

```diff
@@ -11,94 +11,64 @@
+                            <span class="toggle-handle"></span>
+                        </div>
+                    </label>
+                    <p class="tooltip">Hold <em class="modifier-key">shift</em> to scan</p>
+                </div>
             </div>
-        </a>
-        <a tabindex="0" class="nav-button action-open-search" title="Search" data-hotkey='["global:openSearchPage","title","Search ({0})\nShift+click to open here"]'>
-            <span class="icon" data-icon="magnifying-glass"></span>
-        </a>
-        <a tabindex="0" class="nav-button action-open-info" title="Information" data-hotkey='["global:openInfoPage","title","Information ({0})"]'>
-            <span class="icon" data-icon="question-mark-circle"></span>
-        </a>
-    </div>
-</div>
-
-<div id="full">
-    <h3 id="extension-info">Yomitan</h3>
-    <label class="toggle">
-        <input type="checkbox" class="enable-search">
-        <div class="toggle-group">
-            <span class="toggle-on">On</span>
-            <span class="toggle-off">Off</span>
-            <span class="toggle-handle"></span>
         </div>
-    </label>
-    <a tabindex="0" class="link-group action-select-profile">
-        <span class="link-group-icon" data-icon="profile"></span>
-        <span class="link-group-label">Profile</span>
-        <span class="profile-select-container"><select class="profile-select">
-            <optgroup label="Default Profile" class="profile-select-option-group"></optgroup>
-        </select></span>
-    </a>
-    <a tabindex="0" class="link-group action-open-settings">
-        <span class="link-group-icon" data-icon="cog"></span>
-        <span class="link-group-label">Settings</span>
-        <span class="link-group-badge">
-            <div class="flex-margin-left warning-badge no-dictionaries-enabled-warning" hidden><span class="icon" data-icon="exclamation-point-short"></span></div>
-        </span>
-    </a>
-    <a tabindex="0" class="link-group action-open-permissions" hidden>
-        <span class="link-group-icon" data-icon="key"></span>
-        <span class="link-group-label">Permissions</span>
-        <span class="link-group-badge">
-            <div class="flex-margin-left warning-badge permissions-required-warning" hidden><span class="icon" data-icon="exclamation-point-short"></span></div>
-        </span>
-    </a>
-    <a tabindex="0" class="link-group action-open-search">
-        <span class="link-group-icon" data-icon="magnifying-glass"></span><span class="link-group-label">Search</span>
-    </a>
-    <a tabindex="0" class="link-group action-open-info">
-        <span class="link-group-icon" data-icon="question-mark-circle"></span><span class="link-group-label">Information</span>
-    </a>
-</div>
-</div>
-
+        <div tabindex="0" class="action-container">
+            <button type="button" class="low-emphasis action-open-settings" title="Settings" data-hotkey='["global:openSettingsPage","title","Settings ({0})"]'>
```

Comment:
Idk what css element this is, but the blue border that shows up when you hover over the action icons shows up somewhat slow. Could you make the animation faster? If I move across the buttons somewhat fast it feels unresponsive

Reply:
That's because theres an override on the animation duration here https://github.com/yomidevs/yomitan/blob/5f41352fd78cd14468e66e70007a39e2ee20f19c/ext/css/action-popup.css#L41